btrace-baader/device-opcua-go
# Bug/ep 358 cpu usage of b one gateway too high subscription fix

# PR Checklist

> **If your build fails** due to your commit message not passing the build checks, please review the guidelines [here](https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md).

Please check if your PR fulfills the following requirements:



## Testing Instructions

1. Connect to opcua device with S7 using no security( no username, no password, Policy= None, Mode= None)
   - Subscriptions will start ( see device opcua container log
   - disconnect the machine -> subscriptions stop
   - reconnect machine-> subscriptions should start again
2. Connect to opcua device with Simotion using security(username, password, Policy = Basic256Sha256, Mode= SignAndEncrypt)
   - Subscriptions will start ( see device opcua container log
   - disconnect the machine -> subscriptions stop
   - reconnect machine-> subscriptions should start again

## New Dependency Instructions (If applicable)
